### DISCUSSION TITLE
Undefined prediction /api/v1/prediction/undefined

### DISCUSSION THREAD

**Discussion by mejaatg** [FlowiseAI/Flowise#1680] [Q&A]
When we launch a new workflow using an assistant. It doesn't use our own knowledge (ZEP vector database). sometimes it hallucinates. after going through logs under the --DEBUG mode, we saw post requests being  made to the this endpoint: /api/v1/prediction/undefined
could this be the reason or what is there something else that we are missing.

thnanks in advance.

**Comment by HenryHengZJ** [comment]
i dont think thats the issue, if you have undefined the call would return 500
